pkg/scheduler/predicates/ha.go

```diff
@@ -132,7 +132,14 @@ func (h *ha) Filter(instanceName string, pod *apiv1.Pod, nodes []apiv1.Node) ([]
 		}
 
 		podsCount := len(podNames)
-		if podsCount+1 >= int(replicas+1)/2 {
+		// When replicas equals 3, pods on each node should not be greater than 1.
+		maxPodsPerNode := 1
+		if replicas > 3 {
```

Comment:
`3` is hard-coded, but it should be gathered from configuration (in case the user sets it to 5)

Reply:
I think `3` is the number of copies of data regions in TiKV. Is it possible to has more than 3 copies when the replicas of TiKV instances increases, e.g. replicas data regions into more than 3 nodes? cc @tennix @weekface

Reply:
`3` is the tikv replicas.

Reply:
This pr just fix the corner case when replicas is 4.